packages/extension-chakra-storefront/src/components/header/index.jsx

```diff
@@ -197,132 +205,127 @@ const Header = ({
                         <SearchBar />
                     </HideOnMobile>
 
-                    {!isRegistered && (
-                        <IconButtonWithRegistration
-                            aria-label={intl.formatMessage({
-                                id: 'header.button.assistive_msg.my_account',
-                                defaultMessage: 'My Account'
-                            })}
-                            variant="unstyled"
-                            css={{...styles.iconButton, ...styles.accountIconButton}}
-                            onClick={onMyAccountClick}
-                            onMouseOver={isDesktop ? onAccountMenuOpen : noop}
-                        >
-                            <AccountIcon boxSize={6} />
-                        </IconButtonWithRegistration>
-                    )}
+                    <IconButtonWithRegistration
```

Comment:
I am reverting the code we have for v3 because moving this icon into popover trigger causes the navigation to account onSuccessLogin failed.